plugins/main/public/components/common/wazuh-discover/wz-discover.tsx

```diff
@@ -200,19 +206,21 @@ const WazuhDiscoverComponent = (props: WazuhDiscoverProps) => {
         grow
       >
         <>
-          {isLoading ? (
+          {isDataSourceLoading ? (
             <LoadingSpinner />
           ) : (
-            <SearchBar
-              appName='wazuh-discover-search-bar'
-              {...searchBarProps}
-              showSaveQuery={true}
-            />
+            <div className="wz-search-bar">
+              <SearchBar
+                appName='wazuh-discover-search-bar'
+                {...searchBarProps}
+                showSaveQuery={true}
+              />
+            </div>
           )}
-          {!isLoading && results?.hits?.total === 0 ? (
+          {!isDataSourceLoading && results?.hits?.total === 0 ? (
             <DiscoverNoResults timeFieldName={timeField} queryLanguage={''} />
```

Comment:
`LoadingSpinner` and `DiscoverNoResults` are components that are being imported from vulnerabilities. These could be generated/moved to a common place since they will surely be reused in all dashboards.